mypykaizen/mypykaizen.py

```diff
@@ -82,7 +122,7 @@ def main() -> None:
         allowable_errors.mypy_version = mypy.version.__version__
         needs_save = True
 
-    output_lines = result.stdout.splitlines()
+    output_lines = sanitize_output_lines(result.stdout.splitlines())
     last_line = output_lines[-1]
     output_lines = output_lines[:-1]  # Remove the last line which is just the summary
```

Comment:
May be slightly more robust to strip the last line first?

```suggestion
    output_lines = result.stdout.splitlines()
    last_line = output_lines[-1]
    output_lines = output_lines[:-1]  # Remove the last line which is just the summary
    output_lines = sanitize_output_lines(output_lines)
```

Reply:
Eh - you know what - I think this is good as is